apollo-federation/src/sources/connect/json_selection/apply_to.rs

```diff
@@ -201,28 +210,17 @@ impl ApplyTo for NamedSelection {
                         key.dotted(),
                         json_type_name(data),
                     ).as_str(),
```

Comment:
```suggestion
                    ),
```

I guess this change (and the required one from earlier) don't necessarily belong to this PR, so we could always optimize this in a follow-up.